tests/unit-tests/admin/test-class-sensei-setup-wizard.php

```diff
@@ -269,6 +287,17 @@ public function testActivationRedirectWithoutTransient() {
 		$setup_wizard_mock->activation_redirect();
 	}
 
+	/**
+	 * Testing if redirect option is cleared on setup wizard rendering.
+	 */
+	public function testRenderWizardPageClearsRedirectOption() {
+		update_option( 'sensei_activation_redirect', 1 );
+
+		Sensei()->setup_wizard->render_wizard_page();
+
+		$this->assertFalse( get_option( 'sensei_activation_redirect', false ) );
```

Comment:
Maybe we can divide these in Arrange Act Assert sections?

Reply:
https://github.com/Automattic/sensei/commit/2d7b8eb903dbb8dc00b466e502505cda51ce9c79